kin/account.py

```diff
@@ -139,142 +141,128 @@ def get_transaction_history(self, amount=10, descending=True, cursor=None, simpl
         """
 
         return self._client.get_account_tx_history(self.get_public_address(),
-                                                   amount=10,
-                                                   descending=True,
-                                                   cursor=None,
-                                                   simple=True)
+                                                   amount=amount,
+                                                   descending=descending,
+                                                   cursor=cursor,
```

Comment:
how cursor can be used? where you can extract it?

Reply:
the cursor is used when I want to get more than 200 transactions, so I can use it for paging